lib/leanplum_api/configuration.rb

```diff
@@ -29,6 +29,9 @@ class Configuration
     attr_accessor :logger
     attr_accessor :timeout_seconds
 
+    # Override validations for leanplum response. On by default.
+    attr_accessor :validate_response
```

Comment:
curious - why do we want this option?

Reply:
I stub LP responses in our Rails app. VCR won't integrate for 'reasons'. Hence not checking counts of return success vs send items. Only a testing feature.

Reply:
:ok: